internal/testhelpers/e2e/env_windows.go

```diff
@@ -0,0 +1,37 @@
+//go:build windows
```

Comment:
Have you tried running e2e tests locally with these changes? The variables defined here may be too finely-tuned for GitHub Actions CI.

Reply:
Yes, they ran fine on my local machine. We also can disable the sandbox with an env var if it's a problem. I expect there to be some issues with this and we can change it to opt-in as well as fine-tune the sandboxing if/when issues pop up.

Reply:
Oh and as for the env vars on Windows they appear to be fairly standard, except for the msys one which should not only be prepended when on CI.